### DISCUSSION TITLE
Registries supporting ORAS

### DISCUSSION THREAD

**Discussion by orkaplan-M** [oras-project/oras#357] [Q&A] [answered]
Hi, do you know if the following currently support or plan to support (if so when) ORAS?
- JFrog
- DockerHub
- GitLab CR
- Harbor
- RedHat Quay

Thanks,
Or

**Comment by shizhMSFT** [comment, marked as answer]
ORAS supports all OCI registries. You may checkout the [OCI certified registry providers](https://conformance.opencontainers.org/).

Although your listed registries are not certified, they are OCI-compliant registries and thus supported by ORAS.
- JFrog supported.
- DockerHub is based on [distribution](https://github.com/distribution/distribution) and thus supported.
- GitLab CR: seems work (see #260).
- Harbor is OCI-compliant since 2.0 and thus supported.
- Quay is OCI-compliant since 3.3 and thus supported.

**Reply by orkaplan-M** [reply]
Thanks for the answer!
Do you know if GCR is also plan to be supported or only GAR?

**Reply by shizhMSFT** [reply]
I'm not sure about GCR but GAR should work since GAR is OCI compliant.

**Reply by orkaplan-M** [reply]
Do you know which registries are supported by ORAS artifact reference as well?
from what we've found it's only ACR, ECR, Docker Hub and CNCF Distribution

**Reply by shizhMSFT** [reply]
[ORAS artifact reference](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) is a different one. In addition to the registry list you have mentioned, [zot](https://github.com/project-zot/zot) also supports ORAS artifact reference.

@SteveLasker Do you know any other registries implementing [artifact-spec](https://github.com/oras-project/artifacts-spec)?

**Reply by SteveLasker** [reply]
[ORAS Artifacts](https://github.com/oras-project/artifacts-spec) is still in Alpha, and we're excited to see Zot engage early on. 
[AWS ECR has also committed support](https://github.com/aws/containers-roadmap/issues/43#issuecomment-1011605357). As the ORAS Artifacts spec reaches RC, we hope to see others engage as well.

We are also contributing to the [OCI Working Group for Reference Types](https://github.com/opencontainers/wg-reference-types/) to adopt these rich capabilities and encourage folks to provide feedback.

**Reply by orkaplan-M** [reply]
Thanks for the answer!
